CHANGES.rst

```diff
@@ -18,6 +18,8 @@ New Features
 
 - Allow Subset Plugin to edit composite subsets. [#2182]
 
+- Allow secondary data products to be removed from the app (except for Mosviz) [#2194]
```

Comment:
what does "secondary" mean here?  Maybe "Allow user-loaded data to be removed from the app (except for mosviz and cubeviz)"?

Reply:
Yep, your interpretation is correct; it was meant to signify any datasets that aren't the one being actively shown. But I like your language better